javascript/javascript1/week4/class-exercises/card-number-formatter.js

```
@@ -0,0 +1,32 @@
+const groupSize = 4; //'1234 5678 9123 4..'
+
+function formatCreditCardNumber(number) {
+  if (typeof number !== 'number') {
+    console.log('Error: The provided value is not a number!');
+    return;
```

There is logged "undefined" after this console.log. To avoid it, you could just return the error message, instead of logging it. Minor thing.
```
return 'Error: The provided value is not a number!';
```